### PR TITLE
update integration bar and improve email picker

### PR DESCRIPTION
**Pull request by Vadman97**
## Summary

* Updates styles of `/setup/alerts`
* Ensures that the alerts creation flow is only shown once.
* Updates platform picker styles

## How did you test this change?

[Reflame preview](https://preview.highlight.io/1/setup/alerts?%7Er_preview=%7B%22action%22%3A%22start%22%2C%22mode%22%3A%22production%22%2C%22region%22%3A%22sjc%22%2C%22variantId%22%3A%2201H8N0ZTV62YEH2QAV20B1HW8F%22%2C%22variantData%22%3A%22%7B%5C%22type%5C%22%3A%5C%22branch%5C%22%2C%5C%22branch%5C%22%3A%5C%22vadim%2Fimprove-setup-alerts%5C%22%2C%5C%22githubOwnerName%5C%22%3A%5C%22highlight%5C%22%2C%5C%22githubRepositoryName%5C%22%3A%5C%22highlight%5C%22%7D%22%7D).

## Are there any deployment considerations?

No

### CODE REVIEW

File: frontend/src/pages/Setup/SetupRouter/AlertsSetup.tsx [hunk 1]
```diff
@@ -515,25 +518,65 @@ const AlertPicker = function ({
...
+				<Box>
+					<Button
+						trackingId="save-alerts"
+						kind="primary"
+						size="small"
+						emphasis="high"
+						iconRight={<IconSolidCheveronRight />}
+						loading={createLoading.current}
+						disabled={createLoading.current}
+						onClick={async () => {
```
Comment (deltaepsilon) [inline frontend/src/pages/Setup/SetupRouter/AlertsSetup.tsx:531]: This is a lot for an inline onClick. Consider extracting into a separate function.

File: frontend/src/pages/Setup/SetupRouter/AlertsSetup.tsx [hunk 2]
```diff
@@ -515,25 +518,65 @@ const AlertPicker = function ({
...
+							try {
+								createLoading.current = true
+								await createAlerts()
+								message.success(
+									`${alertsSelected.length} alert${
+										alertsSelected.length > 1 ? 's' : ''
+									} updated!`,
+								)
+								await new Promise((r) => {
```
Comment (deltaepsilon) [inline frontend/src/pages/Setup/SetupRouter/AlertsSetup.tsx:540]: We should have a utility function named `wait`, cause this is a very common thing to do.

**Review by deltaepsilon — APPROVED** [review]

File: frontend/src/pages/Setup/SetupRouter/AlertsSetup.tsx (hunk 2 above)
Comment (Vadman97) [reply frontend/src/pages/Setup/SetupRouter/AlertsSetup.tsx]: yeah good point! dropping this change tho per feedback from today's demo session.

File: frontend/src/pages/Setup/SetupRouter/AlertsSetup.tsx (hunk 1 above)
Comment (Vadman97) [reply frontend/src/pages/Setup/SetupRouter/AlertsSetup.tsx]: good idea! cleaning this up

**Pull request merged**